tests/test_MooseFSSR.py

```diff
@@ -0,0 +1,64 @@
+import mock
+import MooseFSSR
+import unittest
+
+
+class FakeMooseFSSR(MooseFSSR.MooseFSSR):
+    uuid = None
+    sr_ref = None
+    srcmd = None
+    other_config = {}
+
+    def __init__(self, srcmd, none):
+        self.dconf = srcmd.dconf
+        self.srcmd = srcmd
+
+
+class TestMooseFSSR(unittest.TestCase):
+
+    def create_moosefssr(self, masterhost='aServer', rootpath='/aServerpath',
+                     sr_uuid='asr_uuid', useroptions=''):
+        srcmd = mock.Mock()
+        srcmd.dconf = {
+            'masterhost': masterhost,
+            'rootpath': rootpath
+        }
+        if useroptions:
+            srcmd.dconf.update({'options': useroptions})
```

Comment:
Python coverage is a bit boring, could you remove these lines? Thx! :slightly_smiling_face: 

```
Test coverage
Name                             Stmts   Miss Branch BrPart  Cover   Missing
----------------------------------------------------------------------------
tests/test_MooseFSSR.py             43      1      2      1    96%   27
```